File: BECore/app/model/ProjectPlanPerform_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectPlanPerform_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectPlanPerform_AttachDetail', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectPlanPerform_AttachDetailID',
    idType: 'Guid',
    fields: [
        { name: "ProjectPlanPerform_AttachDetailID", type: "string" },
		{ name:"ProjectPlanPerformID", type:"string" },
		{ name:"FileName", type:"string" },
		{ name: "FileType", type: "string" },
        { name: "Description", type: "string" },
		{ name:"FileSize", type:"string" },
		{ name:"Inactive", type:"boolean" },
		{ name:"SortOrder", type:"int" },
		{ name:"CreatedDate", type:"date" },
		{ name:"ModifiedDate", type:"date" },
		{ name:"IPAddress", type:"string" },
		{ name:"ModifiedBy", type:"string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
     ]
});

File: MT.Model/Business/ProjectPlanPerform_AttachDetail.cs
﻿using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectPlanPerform_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectPlanPerform_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectPlanPerform_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectPlanPerform_AttachDetailID { get; set; }
        public Guid ProjectPlanPerformID { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public Guid? FileResourceID { get; set; }
        public string Description { get; set; }
        
    }
    #endregion
}

